### PR TITLE
Development

### PR DESCRIPTION
**Pull request by kobiyoyo**
pull request for code review

### CODE REVIEW

File: main.css [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+html {
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
+  box-sizing: border-box;
+}
+*, *:before, *:after {
+  -webkit-box-sizing: inherit;
+  -moz-box-sizing: inherit;
+  box-sizing: inherit;
+  }*/
```
Comment (samgaco) [inline main.css]: It's better to delete commented code.

**Review by samgaco — CHANGES_REQUESTED** [review]
### Hi @kobiyoyo :hand: 

### Great job so far! :clap:  :100: 

* The mobile screen width should go up to 480px, medium from 768px to 1024px and large from 1024px an up.

* The section **top story** should be wrapped in `<section>` tags. The `<article>` tags inside this section should have the width set to 100%.

* The section **more stories** should be enclosed in `<section>` tags. The `<article>` tags inside this section should have the width set to 100%. The small paragraphs in these articles (under the title) should also be set to 100% width. 

* The section **opinions** should be enclosed in `<section>` tags. It should contain 8 articles. _The `<article>` tags inside this section should have the width set to 100%. The small paragraphs in these articles (under the title) should also be set to 100% width._ The author name should be written under the title. The title and the image should have floats set to the left.


* The section **latest news**. The section just underneath opinions is not showing the header. As in the others, The `<article>` tags inside this section should have the width set to 100%. The image should be floated to the left and the title of the article to the right. Both image and title should have the width set to 50%.

* **In the magazine:** Is lacking the Newsweek logo at the left. The `<article>` tags inside this section should have the width set to 100%. The images should be set to 100% too. Each article has a category and a title aligned to the left and below the image

* **Editor's picks**. It's also lacking a logo. The `<article>` tags inside this section should have the width set to 100%. 

* **Featured slideshow** Is also lacking the logo. The `<article>` tags inside this section should have the width set to 100%. 

* You should achieve this arrangement in the footer for the small screen:

> 
footerell

This is how is looking now;

> footertu

* There is a breakpoint in the top stories section in which the section disappears abruptly. Try to fix it.

Alt Text

* The navigation bar elements are a bit aligned to the left. They should be occupying all the space up until the search bar.

### Please :pray:  work on these issues and submit back for another code review. Working on these things will help you grow as a developer, so good spirit and happy coding. :muscle:

**Comment by bolah2009** [comment]
PR currently been reviewed

**Review by bolah2009 — APPROVED** [review]
Nice Job :clap: 

The [clone](https://raw.githack.com/kobiyoyo/BOOTSTRAP/development/index.html) looks good to me. :+1: 

#### You can still work on the responsiveness especially for small screens.

Good luck with your next project. :star2: 

Cheers
Bola

**Pull request merged**